src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -577,13 +576,24 @@ private static boolean canConnectWithPrimaryUser()
     private void doPreamble()
     {
         signIn();
+
+        // Only do this as part of test startup if we haven't already checked. Since we do this as the last
+        // step in the test, there's no reason to bother doing it again at the beginning of the next test
+        if (!_checkedLeaksAndErrors && !"DRT".equals(System.getProperty("suite")))
```

Comment:
I'm curious to know why we plan to behave differently for DRT than for other suites.

Reply:
This is just maintaining existing behavior. The thought is that we want DRTs to run as fast a possible so we only do the error and leak checks once; at the end of the suite.